content-src/asrouter/components/ModalOverlay/ModalOverlay.jsx

```diff
@@ -25,8 +25,8 @@ export class ModalOverlayWrapper extends React.PureComponent {
   render() {
     const {props} = this;
     return (<React.Fragment>
-      <div className="modalOverlayOuter active" onClick={props.onClose} role="presentation" />
-      <div className={`modalOverlayInner active ${props.innerClassName || ""}`}
+      <div className="modalOverlayOuter active" onClick={props.onClose} onKeyDown={this.onKeyDown} role="presentation" />
+      <div className={`${props.innerClassName || "modalOverlayInner active"}`}
```

Comment:
Why did you switch this? Like why not `modalOverlayInner active ${props.innerClassName || ""}`?